js/app.js

```diff
@@ -48,7 +49,7 @@
 
     let lastNotificationTime = 0, noFace = 0;
 
-    const performDetection = async (initial) => {
+    let performDetection = async (initial) => {
```

Comment:
```suggestion
    const performDetection = async (initial) => {
```

Reply:
I made this change before realizing this whole thing was scoped within an anonymous function.

Reply:
@carlin-q-scott I never left a comment on this, can you add a UI input to configure this "break" time and I'll merge the PR and add you ass a contributor?